### PR TITLE
Always display the key counter during replay/autoplay

### LINKED ISSUE #2641: KeyCounterOverlay not always visible for replays
Likely depends on #2621 so `KeyCounterCollection` can DI the current `WorkingBeatmap` to check for an auto mod.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/58859220-keycounteroverlay-not-always-visible-for-replays?utm_campaign=plugin&utm_content=tracker%2F4707269&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F4707269&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by smoogipoo**
Resolves #2641.

### CODE REVIEW

File: osu.Game/Screens/Play/ReplayPlayer.cs [hunk 1]
```diff
@@ -14,6 +15,12 @@ public ReplayPlayer(Replay replay)
             Replay = replay;
         }
 
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            HudOverlay.KeyCounter.Visible.Value = true;
```
Comment (peppy) [inline osu.Game/Screens/Play/ReplayPlayer.cs]: is this necessary? isn't it already handled above?

Comment (peppy) [reply osu.Game/Screens/Play/ReplayPlayer.cs]: what's the one above for then?

Comment (smoogipoo) [reply osu.Game/Screens/Play/ReplayPlayer.cs]: The one in Player itself you mean? That's for Autoplay, since that doesn't go through a ReplayPlayer.

Comment (peppy) [reply osu.Game/Screens/Play/ReplayPlayer.cs]: image

can't we just use the fact that this is already a `Bindable`?

Comment (smoogipoo) [reply osu.Game/Screens/Play/ReplayPlayer.cs]: Indeed that can be done, nice catch.

**Review by peppy — APPROVED** [review]

**Pull request merged**